gazebo_plugins/src/gazebo_ros_joint_pose_trajectory.cpp

```diff
@@ -295,8 +295,8 @@ bool GazeboRosJointPoseTrajectory::SetTrajectory(
   this->disable_physics_updates_ = req.disable_physics_updates;
   if (this->disable_physics_updates_)
   {
-    this->physics_engine_enabled_ = this->world_->GetEnablePhysicsEngine();
-    this->world_->EnablePhysicsEngine(false);
+    this->physics_engine_enabled_ = this->world_->SetPhysicsEnabled();
```

Comment:
either do as before : +    this->world_->SetPhysicsEnabled(false);
 +    this->physics_engine_enabled_ = false;

Or this->physics_engine_enabled_ = this->world_->PhysicsEnabled();
  this->world_->SetPhysicsEnabled(false);